roles/sat6repro/vars/main.yml

```diff
@@ -29,3 +29,6 @@ required_root_free_space: 75
 
 # Disable firewall - This setting can optionally be used to disable firewall to setup Satellite for testing purposes
 disable_firewall: false
+
+# Run katello-reindex - This setting can optionally be used to run katello reindex after the Satellite install.
```

Comment:
You may want to add a note that reindex will be run in a 6.1>6.2 upgrade - in case a user wants to clone then upgrade immediately after, this will save them some time